pytest_reana/fixtures.py

```diff
@@ -440,7 +440,7 @@ def test_publish_hello(default_in_memory_producer, default_queue):
         serializer='json')
 
 
-@pytest.fixture(scope='module')
+@pytest.fixture
```

Comment:
Why do we change the scope? Did we find trouble already? 😟

Reply:
Actually this doesn't have to be changed, I'll add it back. It is `module` even without specifying it, because of `tmp_shared_volume_path`, but better to read it here too.